src/main/web/src/organisms/login/Login.tsx

```diff
@@ -121,15 +121,17 @@ const Login: React.FC<Props> = () => {
                             </div>
 
                             <div className="google-oauth-login">
```

Comment:
Google OAuth Provider is not necessary here, it is already used in App.tsx

Reply:
I have not added but only taken over